src/main/java/uk/ac/ebi/eva/pipeline/parameters/JobOptions.java

```diff
@@ -113,186 +65,49 @@ public void loadArgs() throws IOException {
-
-        // TODO This block shall be removed when we use Spring Data parameters only
-        variantOptions.put(VariantStorageManager.VARIANT_SOURCE, source);
-        variantOptions.put(VariantStorageManager.OVERWRITE_STATS, overwriteStats);
-        variantOptions.put(VariantStorageManager.INCLUDE_SRC, includeSourceLine);
-        variantOptions.put("compressExtension", compressExtension);
-        variantOptions.put(VariantStorageManager.ANNOTATE, annotate);
-        variantOptions.put(VariantStatisticsManager.BATCH_SIZE, chunkSize);
-
-        variantOptions.put(VariantStorageManager.DB_NAME, dbName);
-        variantOptions.put(MongoDBVariantStorageManager.OPENCGA_STORAGE_MONGODB_VARIANT_DB_NAME, dbName);
-        variantOptions.put(MongoDBVariantStorageManager.OPENCGA_STORAGE_MONGODB_VARIANT_DB_HOSTS, dbHosts);
-        variantOptions.put(MongoDBVariantStorageManager.OPENCGA_STORAGE_MONGODB_VARIANT_DB_AUTHENTICATION_DB, dbAuthenticationDb);
-        variantOptions.put(MongoDBVariantStorageManager.OPENCGA_STORAGE_MONGODB_VARIANT_DB_USER, dbUser);
-        variantOptions.put(MongoDBVariantStorageManager.OPENCGA_STORAGE_MONGODB_VARIANT_DB_PASS, dbPassword);
-        // TODO End of block to be removed when we use Spring Data parameters only
-
-        logger.debug("Using as input: {}", input);
-        logger.debug("Using as variantOptions: {}", variantOptions.entrySet().toString());
-    }
-
     private void loadPipelineOptions() {
-        pipelineOptions.put(JobParametersNames.INPUT_VCF, input);
-        pipelineOptions.put("compressExtension", compressExtension);
-        pipelineOptions.put(JobParametersNames.OUTPUT_DIR, outputDir);
-        pipelineOptions.put(JobParametersNames.OUTPUT_DIR_STATISTICS, outputDirStatistics);
-        pipelineOptions.put(JobParametersNames.INPUT_PEDIGREE, pedigree);
-        pipelineOptions.put(JobParametersNames.INPUT_GTF, gtf);
-        pipelineOptions.put(JobParametersNames.DB_NAME, dbName);
-        pipelineOptions.put(JobParametersNames.DB_COLLECTIONS_VARIANTS_NAME, dbCollectionVariantsName);
-        pipelineOptions.put(JobParametersNames.DB_COLLECTIONS_FILES_NAME, dbCollectionFilesName);
-        pipelineOptions.put(JobParametersNames.DB_COLLECTIONS_FEATURES_NAME, dbCollectionGenesName);
-        pipelineOptions.put(JobParametersNames.DB_COLLECTIONS_STATISTICS_NAME, dbCollectionStatsName);
-        pipelineOptions.put(JobParametersNames.CONFIG_DB_HOSTS, dbHosts);
-        pipelineOptions.put(JobParametersNames.CONFIG_DB_AUTHENTICATIONDB, dbAuthenticationDb);
-        pipelineOptions.put(JobParametersNames.CONFIG_DB_USER, dbUser);
-        pipelineOptions.put(JobParametersNames.CONFIG_DB_PASSWORD, dbPassword);
-        pipelineOptions.put(JobParametersNames.CONFIG_DB_READPREFERENCE, readPreference);
         pipelineOptions.put(JobParametersNames.ANNOTATION_SKIP, skipAnnot);
         pipelineOptions.put(JobParametersNames.STATISTICS_SKIP, skipStats);
```

Comment:
Skip Annot and Skip Stats not yet, I'm checking if I can do it and I'll send it in another PR